docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.9"
+services:
+  srv:
+    container_name: http_server
+# v1: Using dockerfile together with image is not allowed,
+# using build together with image is not allowed.
+# Attempting to do so results in an error.  
+#    dockerfile: Dockerfile
+    build:
+      context: .
+      cache_from:
+        - node:10-alpine
+    image: static_srv:alpine
+    env_file: .env
+    ports:
+      - "${EXT_PORT}:${PORT}"
```

Comment:
verify whether $PORT can be changed after container-image was built and that newly set port still works if changed?